inc/cachify.class.php

```diff
@@ -473,28 +473,14 @@ private static function _get_options() {
 	 * Modify robots.txt
 	 *
 	 * @since   1.0
-	 * @change  2.1.9
+	 * @change  2.4.0
 	 *
-	 * @param   string $data  Original content of dynamic robots.txt.
-	 * @return  string        Modified content of robots.txt.
 	 */
-	public static function robots_txt( $data ) {
+	public static function robots_txt() {
 		/* HDD only */
-		if ( self::METHOD_HDD !== self::$options['use_apc'] ) {
-			return $data;
+		if ( self::METHOD_HDD === self::$options['use_apc'] ) {
+			echo 'Disallow: */cache/cachify/';
 		}
-
-		/* Parse site URL */
-		$url_parts = wp_parse_url( site_url() );
-
-		/* Output */
-		$data .= sprintf(
-			'%2$sDisallow: %1$s/wp-content/cache/cachify/%2$s',
```

Comment:
Should this respect the `WP_CONTENT_DIR` constant?

Reply:
For the documentation we just changed to `*` (#194), but in this point we _could_ be more precise.

Reply:
we can use
printf(
	'Disallow: *%s/cache/cachify/',
	wp_make_link_relative( content_url() )
);
if you want. But when you move wp-content to another location or rename it then you don't wan't to leak it's path to public.